Setup/UpgradeSchema.php

```diff
@@ -99,42 +98,61 @@ public function upgrade(
                 $setup->getTable('quote'),
                 'delivery_options',
                 [
-                    'type' => Table::TYPE_TEXT,
+                    'type'     => Table::TYPE_TEXT,
                     'nullable' => true,
-                    'comment' => 'MyParcel delivery options',
+                    'comment'  => 'MyParcel delivery options',
                 ]
             );
 
             $setup->getConnection()->addColumn(
                 $setup->getTable('sales_order'),
                 'delivery_options',
                 [
-                    'type' => Table::TYPE_TEXT,
+                    'type'     => Table::TYPE_TEXT,
                     'nullable' => true,
-                    'comment' => 'MyParcel delivery options',
+                    'comment'  => 'MyParcel delivery options',
                 ]
             );
 
             $setup->getConnection()->addColumn(
                 $setup->getTable('sales_order'),
                 'drop_off_day',
                 [
-                    'type' => Table::TYPE_DATE,
+                    'type'     => Table::TYPE_DATE,
                     'nullable' => true,
-                    'comment' => 'MyParcel drop off day',
+                    'comment'  => 'MyParcel drop off day',
                 ]
             );
             $setup->getConnection()->addColumn(
                 $setup->getTable('sales_order_grid'),
                 'drop_off_day',
                 [
-                    'type' => Table::TYPE_DATE,
+                    'type'     => Table::TYPE_DATE,
                     'nullable' => true,
-                    'comment' => 'MyParcel drop off day',
+                    'comment'  => 'MyParcel drop off day',
+                ]
+            );
+        }
+        if (version_compare($context->getVersion(), '2.4.3', '<=')) {
```

Comment:
```suggestion
        if (version_compare($context->getVersion(), '3.0.0', '<')) {
```
Want anders kunnen we geen 2.4.4 meer maken. Als iemand na 2.4.4 naar 3.x gaat, dan krijgt hij niet meer deze update.

Reply:
Done